src/__CPlugin.ino

```diff
@@ -18,6 +18,15 @@ static const char ADDCPLUGIN_ERROR[] PROGMEM = "System: Error - To much C-Plugin
 */
 #define ADDCPLUGIN(NNN) if (x < CPLUGIN_MAX) { CPlugin_id[x] = CPLUGIN_ID_##NNN; CPlugin_ptr[x++] = &CPlugin_##NNN; } else addLog(LOG_LEVEL_ERROR, FPSTR(ADDCPLUGIN_ERROR));
 
+void CPluginConnected(void)
+{
+    CPluginCall(CPLUGIN_CONNECTED, 0);
+}
+
+void CPluginDisonnect(void)
+{
+    CPluginCall(CPLUGIN_DICONNECT, 0);
```

Comment:
Typo in `CPLUGIN_DICONNECT`

Reply:
Corrected. Still need to find a good place to call this function. I think it is only suitable when the "unit name" is changed by the user to inform the controller that the old name is not valid anymore.

Reply:
I think if any of the parameters you want to send in the discovery message is changed, you must send the message again.
And if it is about re-informing the changes, I think you should name it like that.